distribution/test/itests/test-itests-common/pom.xml

```diff
@@ -55,19 +55,19 @@
         <dependency>
             <groupId>org.apache.felix</groupId>
             <artifactId>org.apache.felix.framework</artifactId>
-            <version>4.2.1</version>
+            <version>5.4.0</version>
```

Comment:
Any specific reason we didn't create properties for this and the `org.osgi.*` ones?